package.json

```diff
@@ -54,6 +54,6 @@
     "mocha-jsdom": "^2.0.0",
     "pump": "^3.0.0",
     "sinon": "^9.0.0",
-    "typescript": "3.0.1"
+    "typescript": "^4.2.4"
```

Comment:
just as a sanity check, were there any issues we encountered as a result of moving to a new major version of TS or anything we need to worry about re: compatibility?

Reply:
I don't believe we are affected by any of the following breaking changes: https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#breaking-changes